test-common/test-common-runtime/src/main/java/org/apache/kafka/common/test/ClusterInstance.java

```diff
@@ -288,69 +339,22 @@ default void deleteTopic(String topicName) throws InterruptedException {
     void waitForReadyBrokers() throws InterruptedException;
 
     default void waitForTopic(String topic, int partitions) throws InterruptedException {
```

Comment:
Perhaps it should be renamed to `waitTopicCreation` to correspond with `waitTopicDeletion`.